package/network/services/hostapd/files/hostapd.sh

```diff
@@ -484,7 +485,15 @@ hostapd_set_bss_options() {
 		;;
 		psk|sae|psk-sae)
 			json_get_vars key wpa_psk_file
-			if [ ${#key} -eq 64 ]; then
+			if [ "$auth_type" == "psk" ] && [ "$ppsk" -ne 0 ] ; then
```

Comment:
Please use single "=" instead of bash "=="

Reply:
Done